site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -6,30 +6,84 @@ updated: 2018-02-25
 description: An overview of the purpose of Chrome Extensions and how they're developed.
```

Comment:
(random comment spot)

On the netlify preview, the link from https://deploy-preview-411--developer-chrome-com.netlify.app/docs/extensions/mv3/ to "what are extensions" is broken, but it looks fine in production.  Is that an issue with this PR, or with netlify?  Or where the link is technically wrong, but the production server catches and redirects?  (That's my closest guess?)

I can't see how this PR would have broken it, since it's not moving the file, but figured I'd raise it.

Reply:
Yeah, the link is wrong and redirecting in prod and on localhost but not in the preview. (Ran into this before where netlify doesn't do our redirects.) Fixed the links for good form.